### ISSUE TITLE
K9MAIL_INTERNAL_OUTBOX ist lost after export import cycle

### ISSUE THREAD

**Issue by morckx** [thunderbird/thunderbird-android#3786]
### Expected behavior
After export of settings and accounts, clear data, and import settings and accounts, sending emails should work as before.

### Actual behavior
Sending emails fails silently with:
```
12-02 21:48:19.965 24174 24495 E MessagingController: Error sending message
12-02 21:48:19.965 24174 24495 E MessagingController: com.fsck.k9.mail.MessagingException: LocalFolder.open(): Folder not found: K9MAIL_INTERNAL_OUTBOX (-1)
12-02 21:48:19.965 24174 24495 E MessagingController:   at com.fsck.k9.mailstore.LocalFolder$1.doDbWork(LocalFolder.java:186)
12-02 21:48:19.965 24174 24495 E MessagingController:   at com.fsck.k9.mailstore.LocalFolder$1.doDbWork(LocalFolder.java:165)
12-02 21:48:19.965 24174 24495 E MessagingController:   at com.fsck.k9.mailstore.LockableDatabase.execute(LockableDatabase.java:274)
12-02 21:48:19.965 24174 24495 E MessagingController:   at com.fsck.k9.mailstore.LocalFolder.open(LocalFolder.java:165)
12-02 21:48:19.965 24174 24495 E MessagingController:   at com.fsck.k9.controller.MessagingController.sendMessage(MessagingController.java:1450)
12-02 21:48:19.965 24174 24495 E MessagingController:   at com.fsck.k9.activity.MessageCompose$SendMessageTask.doInBackground(MessageCompose.java:1411)
12-02 21:48:19.965 24174 24495 E MessagingController:   at com.fsck.k9.activity.MessageCompose$SendMessageTask.doInBackground(MessageCompose.java:1382)
12-02 21:48:19.965 24174 24495 E MessagingController:   at android.os.AsyncTask$2.call(AsyncTask.java:333)
12-02 21:48:19.965 24174 24495 E MessagingController:   at java.util.concurrent.FutureTask.run(FutureTask.java:266)
12-02 21:48:19.965 24174 24495 E MessagingController:   at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:245)
12-02 21:48:19.965 24174 24495 E MessagingController:   at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
12-02 21:48:19.965 24174 24495 E MessagingController:   at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
12-02 21:48:19.965 24174 24495 E MessagingController:   at java.lang.Thread.run(Thread.java:764)
```

### Steps to reproduce
1. export settings and accounts
2. clear data
3. import settings and accounts
4. try to send mail

### Environment
K-9 Mail version: 1552c151a565f5ff44a46d2becd35c9033d6bb3b

Android version: 9.0

Account type (IMAP, POP3, WebDAV/Exchange): IMAP

**Comment by artbristol** [comment]
@cketti I managed to reproduce this bug on an old build, but I'm struggling to find the settings import/export button in the new UI - I guess that depends on #3751 ?

**Comment by morckx** [comment]
Yes, OUTBOX now survives export, clear data, import. However a new issue was introduced in the mean time: https://github.com/k9mail/k-9/commit/58717266c03d4a8290f9297b3eabe84f427d6f4d#r31767816

**Comment by morckx** [comment]
Fixed via #3812

**Comment by francwalter** [comment]
> @cketti I managed to reproduce this bug on an old build, but I'm struggling to find the settings import/export button in the new UI - I guess that depends on #3751 ?

@artbristol I use the 5.700 Snapshot and miss that import an export there too, is that fixed in the 5.800?
Thanks, frank
